gigantum/gigantum-client
# Update packages and base image to Ubuntu 20.04

## Pull Request for Issue #1711

**Describe changes introduced by this PR (Bugs fixed, features added, docs updated, etc.)**:
This PR updates our base container to ubuntu 20.04 and updates a bunch of packages to deal with security warnings. 

**Please check if the PR fulfills these requirements**

  - PR is set to merge to integration
  - PR title is descriptive, as this becomes the commit message on squash and merge.
  - Any related issues are added to the extended commit message during squash and merge.
  - Tests have been added (for bug fixes/features)
  - Docs have been added/updated (for bug fixes/features)

**QA Test Steps and Exit Criteria**:
- No functionality change expected
- Normal QA, test framework, and release test plan check-off required.
